File: Java/SpaceVIL/spacevil/src/main/java/com/spvessel/DialogWindow.java
package com.spvessel;

public abstract class DialogWindow extends CoreWindow {

    /**
     * Constructs a DialogWindow
     */
    public DialogWindow() {
        initWindow();
    }

    private String _title = "";

    /**
     * DialogWindow title text
     */
    public String getDialogTitle() {
        return _title;
    }
    public void setDialogTitle(String value) {
        _title = value;
    }
}

File: spacevil_21052019/net_src/Windows/DialogWindow.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SpaceVIL
{
    public abstract class DialogWindow : CoreWindow
    {
        /// <summary>
        /// Constructs a DialogWindow
        /// </summary>
        public DialogWindow()
        {
            InitWindow();
        }

        String _title = String.Empty;

        /// <summary>
        /// DialogWindow title text
        /// </summary>
        public String DialogTitle
        {
            get
            {
                return _title;
            }
            set
            {
                _title = value;
            }
        }
    }
}
